demo/src/API.elm

```diff
@@ -93,16 +98,12 @@ router conn =
             , Cmd.none
             )
 
+        ( GET, Number ) ->
+            ( Conn.interop (GetRandom 10) conn
+            , Cmd.none
+            )
```

Comment:
Here the interop function is being called. Not super happy about the side-effect being put into the conn, but it was the cleanest syntax. It also opens up interop to middleware (i.e. `Conn -> Conn` functions), which could either be seen as a good thing, or a bad thing.